js/script.js

```diff
@@ -0,0 +1,154 @@
+(() => {
+  const ui = {
+    form: document.querySelector(".form"),
+    fields: document.querySelectorAll(".input-field"),
+    nameInput: document.getElementById("name"),
+    emailInput: document.getElementById("email"),
+    messageInput: document.getElementById("message"),
+    submitButton: document.getElementById("submit-button"),
+    container: document.querySelector(".container"),
+  };
+
+  const validateForm = () => {
+    let errors = 0;
+    const nameRegexp = /[a-zA-Z\-'\s]+/;
+    const emailRegexp = /^[A-z0-9.-]{1,}@\w+\.[A-z]{2,3}(\.[a-z]{2})?$/;
+    const msgRegexp = /.*\S.*/;
+
+    const validateField = (regExp, field) => {
+      if (regExp.test(field.value)) {
+        field.classList.remove("is-danger");
+        field.nextElementSibling.classList.add("is-hidden");
+      } else {
+        field.classList.add("is-danger");
+        field.nextElementSibling.classList.remove("is-hidden");
+        errors++;
```

Comment:
why do you need to count errors? you can just disable the button if any of the fields are invalid.